### ISSUE TITLE
merging into pytest-core?

### ISSUE THREAD

**Issue by hpk42** [eisensheng/pytest-catchlog#8]
hello -- i am considering merging capturelog/catchlog functionality into core, see https://mail.python.org/pipermail/pytest-dev/2015-September/003159.html  -- would be great if you participate in the discussion.

**Comment by The-Compiler** [comment]
We already had some discussions in #4 where we planned to move it to `pytest-dev` after #7 is done. #7 is the PR which improves a lot, but also changes the API in an incompatible way - I already [commented there](https://github.com/eisensheng/pytest-catchlog/pull/7#issuecomment-143120895) about having a compatibility layer, which I guess would be needed for merging it in.

I'm not sure if I personally would like to see it merged, actually. It'd mean I'd have to wait longer for new features compared to an independent plugin release cycle, and I have some contributions in mind :wink:

**Comment by bersace** [comment]
Any updates on this topic ?

**Comment by twmr** [comment]
I would also be interested if this proejct were merged into pytest-core. 

Now that #7 is done and the pytest-dev maintainers are in favor of the merge, we can finally start to merge it. Or shouldn't we?

**Comment by twmr** [comment]
> I'm not sure if I personally would like to see it merged, actually. It'd mean I'd have to wait longer for new features compared to an independent plugin release cycle, and I have some contributions in mind :wink:

This is true, but we could merge it into the core but still develop it outside, like what is done for [org-mode](http://www.orgmode.org) in emacs. Org-mode is merged from time to time into emacs core, but developed in a separate repository. I'm not sure if it's worth the effort, given that the current pytest release cylce is approximately one release per month.

@The-Compiler would you like a faster release cylce of pytest-dev than one release per month?

**Comment by The-Compiler** [comment]
No, I'm guessing that would be fine indeed.

**Comment by The-Compiler** [comment]
pytest-catchlog now got merged into the pytest core: https://github.com/pytest-dev/pytest/pull/2794

I'll take care of some obvious remaining issues here, but I'd be glad if someone could help sorting the rest out: https://github.com/pytest-dev/pytest/issues/3003
